schema/feature.d.ts

```diff
@@ -18,6 +23,7 @@ type SubFeature<VersionType> = {
     targets?: {
         variantKey?: string;
```

Comment:
Should we also flesh out the additional supported targets mentioned in https://app.asana.com/0/481882893211075/1208204385124443/f?

Reply:
Yeah, good shout. Done.